client/checks.go

```diff
@@ -88,6 +88,15 @@ type CheckInfo struct {
 	// the check is inactive.
 	Status CheckStatus `json:"status"`
 
+	// Successes is the number of times this check has succeeded. It is reset
+	// when the check succeeds again after the check's failure threshold was
+	// reached. This will be zero if the check has never run, or has never run
+	// successfully.
```

Comment:
Also zero if it has been stopped and then started again (as is "failures"). So stopping and then starting again looks the same as "never run". I think it's probably fine to consider a stop/start sequence as resetting, but I do think we should document that somewhere.

Reply:
Good point - updated the comment.